tests/clims/models/test_substance.py

```diff
@@ -15,6 +17,42 @@ def register_gemstone_type(self):
         return self.register_extensible(GemstoneSample)
 
 
+class SubstancePropertiesTestCase(TestCase):
+
+    def test_can_create_substance_with_properties(self):
+        from clims.services.substance import SubstanceBase, FloatField
+        from clims.models.plugin_registration import PluginRegistration
+        from sentry.models.organization import Organization
+
+        org = Organization.objects.get(name="lab")
+
+        example_plugin, _ = PluginRegistration.objects.get_or_create(
+            name='clims.example.plugin', version='1.0.0', organization=org)
+
+        class ExampleSample(SubstanceBase):
+            moxy = FloatField("moxy")
+            cool = FloatField("cool")
+            erudite = FloatField("erudite")
+
+        moxy = random.randint(1, 100)
+        cool = random.randint(1, 100)
+        erudite = random.randint(1, 100)
+
+        self.app.extensibles.register(example_plugin, ExampleSample)
```

Comment:
FYI: It's also possible to write `self.register_extensible` directly. Then you don't have to create the org or plugin.

Reply:
Neat. I will go back and use these fixtures.